libs/flags/json_flag.go

```diff
@@ -33,7 +35,27 @@ func (j *JsonFlag) Unmarshal(v any) error {
 	if j.raw == nil {
 		return nil
 	}
-	return json.Unmarshal(j.raw, v)
+
+	dv, err := jsonloader.LoadJSON(j.raw)
```

Comment:
Could we do the same for the YAML flag?

I believe that's the only reason we still depend on `github.com/ghodss/yaml`. This package uses the `json` tags in the types as opposed to `yaml` tags, which the upstream YAML package uses. Since we use the `json` tags as well in the `dyn` package it should be possible to replace it.

Not blocking for this PR, of course.